src/app/static/src/tests/components/adr/selectDataset.test.ts

```diff
@@ -982,6 +990,14 @@ describe("select dataset", () => {
         expect(spy.mock.calls[0][0]).toBe(pollingId);
     });
 
+    it("stops polling dataset metadata on newDatasetRelease", () => {
+        const rendered = shallowMount(SelectDataset, {store: getStore()});
+        const pollingId = (rendered.vm as any).pollingId;
+        const spy = jest.spyOn(window, "clearInterval");
+        rendered.setData({newDatasetRelease: {"test": "DATA"} as any})
```

Comment:
This doesn't reflect the way in which the widget is actually used - after choosing a release the user selects "import", at which point you start polling for changes again. In general tests should avoid using the `setData` method and use user actions (e.g.  `input.setValue("v1")`, `button.trigger("click")` etc) and replicate actual user behaviour